src/darwin.js

```diff
@@ -158,6 +158,28 @@ function patchInfoPlist(opts) {
 	return es.duplex(input, es.merge(output, icons));
 }
 
+function createEntitlementsPlist(opts) {
+	const input = es.through();
```

Comment:
I know it's 2020, but this library hasn't been updated to ES6. Please use `var` for now.